purple-pill/js/controllers/fontexportController.js

```diff
@@ -32,140 +112,107 @@ function($scope, $http, sharedScope, $timeout) {
             return;
 
         //Also avoid triggering a new export while we're still not finished
-        if (export_is_running)
+        if (exportIsRunning)
             return;
 
-        export_is_running = true;
-        resetProgressBar();
-
-        function zero_padding(value){
-            return value < 10 ? "0" + String(value) : String(value);
-        }
-
-        function get_timestamp(){
-            var year, month, day, hours, minutes, seconds
-              , date = new Date()
-              ;
-            year = zero_padding(date.getFullYear());
-            month = zero_padding(date.getMonth() + 1); //getMonth() returns a zero-based value: 0=January, 11=December
-            day = zero_padding(date.getDate());
-            hours = zero_padding(date.getHours());
-            minutes = zero_padding(date.getMinutes());
-            seconds = zero_padding(date.getSeconds());
-
-            return [year, month, day].join("") + "-" + [hours, minutes, seconds].join("");
-        }
+        const UI_UPDATE_TIMESLICE = 50; // msecs
 
         var bundle = new $scope.data.stateless.JSZip()
-          , bundleFolderName = "metapolator-export-" + get_timestamp()
-          , bundle_filename = bundleFolderName + ".zip"
+          , bundleFolderName = "metapolator-export-" + getTimestamp()
+          , bundleFileName = bundleFolderName + ".zip"
           , bundleFolder = bundle.folder(bundleFolderName)
-          , bundle_data
-          , instances_for_export = Array()
+          , bundleData
```

Comment:
you don't need this anymore. also, you don't need to `delete bundleData` in `setDownloadBlobLink`